docker-entrypoint.sh

```diff
@@ -56,6 +58,27 @@ copy_plugins_if_required() {
   done
 }
 
+setup_usrlib() {
+  if [ -z "${USER_ARTIFACTS}" ]; then
+    return 0
+  fi
+
+  echo "Setting up usrlib"
+  for user_artifact in $(echo "${USER_ARTIFACTS}" | tr ';' ' '); do
+    echo "Adding artifact ${user_artifact} to usrlib directory."
+
+    mkdir -p "${FLINK_HOME}/usrlib"
```

Comment:
```suggestion
  mkdir -p "${FLINK_HOME}/usrlib"
  for user_artifact in $(echo "${USER_ARTIFACTS}" | tr ';' ' '); do
    echo "Adding artifact ${user_artifact} to usrlib directory."

```
moving `mkdir -p "${FLINK_HOME}/usrlib"` out of loop